### PR TITLE
add mention to the university or similar to intro paragraph in README

### PR DESCRIPTION
**Pull request by avallecam**
As suggested in https://github.com/epiverse-trace/epiparameter/issues/110#issuecomment-1464346371

### CODE REVIEW

**Comment by jamesmbaazam** [comment]
I would suggest adding a sentence to acknowledge cases where a package was developed prior to Epiverse TRACE, and the Initiative has now taken over maintenance. An example is [bpmodels](https://github.com/epiverse-trace/bpmodels).

**Comment by Bisaloo** [comment]
> But, I'm wondering if adding these sentences (both from the author and current maintainer) is sustainable in the long term.

Good point. I personally wouldn't add these sentences to the `README` as I believe they don't add any valuable information about the package functionality and these elements are already stored elsewhere (`DESCRIPTION`, hosting GitHub organization, etc.)

But, if maintainers want to have this in the `README`, then we should try to standardize the phrasing as much as possible to make the `README` easier to parse.

**Review by Bisaloo — APPROVED** [review]

**Comment by ntorresd** [comment]
I think the template is very concise and clear. I would just suggest to add a section either explaining how to report bugs or linking to a specific site that expands on the subject.

File: README.Rmd [hunk 1]
```diff
@@ -22,6 +22,8 @@ knitr::opts_chunk$set(
 
 {{ packagename }} provides functions to ....
 
+{{ packagename }} is developed at the [CENTER|similar](url) at the [UNIVERSITY|similar](url) as part of the [Epiverse-TRACE](https://data.org/initiatives/epiverse/).
+
```
Comment (Bisaloo) [inline README.Rmd:26]:
```suggestion
<!-- This sentence is optional and can be removed -->
{{ packagename }} is developed at the [CENTER|similar](url) at the [UNIVERSITY|similar](url) as part of the [Epiverse-TRACE](https://data.org/initiatives/epiverse/).

```

**Comment by Bisaloo** [comment]
Thanks

**Comment by avallecam** [comment]
> I would just suggest to add a section either explaining how to report bugs

@ntorresd good point! I just used the template and after looking at the dev-utils.R file, it reminded me to use `goodpractice::gp()` that reminds me to add the BugReport field in DESCRIPTION.

https://github.com/epiverse-trace/packagetemplate/blob/d36122f932f091b150c7066bb6a20f5cc6a11e91/R/dev-utils.R#L6

```
It is good practice to
  ✖ add a "BugReports" field to DESCRIPTION
```

After it, editing the `bug_report.md` as an [issue template](https://github.com/epiverse-trace/packagetemplate/tree/main/.github) help to provide more guidance about it.

**Pull request merged**